Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -639,7 +639,7 @@ public void OnPointerDragged(MixedRealityPointerEventData eventData)
         }
 
         /// <inheritdoc />
-        public void OnPointerUp(MixedRealityPointerEventData eventData)
```

Comment:
IMixedRealityPointerHandler also contains OnPointerClicked, can we also make this virtual for consistency?

Reply:
Definitely! Along the same lines, I was wondering if we should make Awake virtual, as well, to follow suit with Start. But maybe those two aren't as tightly "grouped" as the IMixedRealityPointerHandler funcs...

Reply:
I would leave Awake as non-virtual for now since it is reserved for the initialization.